README.md

```diff
@@ -75,8 +75,10 @@ Or with anaconda/conda:
 
 For a development installation (requires npm),
 
-    $ git clone https://github.com/maartenbreddels/ipyvolume.git
-    $ cd ipyvolume
+    $ git clone https://github.com/maartenbreddels/ipyvolume.git	
+    $ cd ipyvolume/js
+	$ nmp install
+	$ cd ..
```

Comment:
No need to do the npm install manually, the setup.py should do this, or did it somehow give you issues?

Reply:
I really don't understand how npm and javascript works but when I tried to install using setup.py it failed.
I found this solution running npm before to create javascript lib before running setup

Reply:
Ok, that is odd, could you open an issue for that and remove this part?